### PR TITLE
style: tidy up comments for ReactiveBinding and command binding

### PR DESCRIPTION
**Pull request by olevett**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Tidying comments in a couple of binding related files.

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [x] The commit follows our guidelines: https://github.com/reactiveui/reactiveui#contribute

**Other information**:

### CODE REVIEW

File: src/ReactiveUI/ReactiveBinding.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public interface IReactiveBinding<TView, TViewModel, TValue> : IDisposable
         where TView : IViewFor
     {
         /// <summary>
-        /// The instance of the view model this binding is applied to.</param>
```
Comment (kentcb) [inline src/ReactiveUI/ReactiveBinding.cs]: lolwat - good catch!

File: src/ReactiveUI/ReactiveBinding.cs [hunk 2]
```diff
@@ -150,9 +141,6 @@ internal class ReactiveBinding<TView, TViewModel, TValue> : IReactiveBinding<TVi
         /// </value>
         public BindingDirection Direction { get; private set; }
 
-        /// <summary>
-        /// Releases unmanaged and - optionally - managed resources.
-        /// </summary>
         public void Dispose()
```
Comment (kentcb) [inline src/ReactiveUI/ReactiveBinding.cs]: Wow, where did this originate from? Looks like some incredibly useful ghostdoc vomit.

Comment (olevett) [inline src/ReactiveUI/ReactiveBinding.cs]: I don't really know...it seemed a bit like stating the obvious. To be honest, I'm not sure any of these comments make sense on the `internal` implementation of the fairly well commented interface...

Comment (ghuntley) [inline src/ReactiveUI/ReactiveBinding.cs]: GhostDoc noise can burn in a pit of :fire:

**Pull request merged**